### DISCUSSION TITLE
Plot multiple optimisation history plots in the same graph?

### DISCUSSION THREAD

**Discussion by syedzayyan** [optuna/optuna#4991] [Q&A] [answered]
Hello !

I have a few SQLIte3 databases of a few different models. I was wondering if I could plot the optimisation history of all these models. The visualisation api only allows one study at a time. But I wanted to compare multiple studies if that makes sense. Could we get data for the history of the plots in a clean manner for plots that discard failed studies for example?

Thanks

**Comment by nzw0301** [comment, marked as answer]
> The visualisation api only allows one study at a time. 

No, https://optuna.readthedocs.io/en/stable/reference/visualization/generated/optuna.visualization.plot_optimization_history.html#optuna-visualization-plot-optimization-history can take list of `study` instances, https://github.com/optuna/optuna/pull/2807's description gives concrete figs.
